### DISCUSSION TITLE
Cant mirror values

### DISCUSSION THREAD

**Discussion by kfishermail** [j-kaltes/Juggluco#68] [Q&A]
I have 2 android phones running juggluco: "BLUE" and "WHITE"

I have WHITE configured to send values to BLUE
I have BLUE configured to recv values from WHITE

On WHITE I click on "new amount" and enter a value for fast insulin.  The value appears immediately on WHITE's JG display.  It does not appear on BLUE.

On BLUE, if I tap "List" command,  it shows greyed-out "Deleted" lines on the left portion of the screen.

To me, it appears the value is being xferred from WHITE to BLUE, Then BLUE is deleting it for some reason

Any ideas?  Could this be related to the value "labels"?

**Comment by j-kaltes** [comment]
Can you show screenshots of the mirror overview and specific connection on both sites?

**Reply by kfishermail** [reply]
Re-installed Juggluco, now it works? wierd?
